src/fhirdefs/load.ts

```diff
@@ -25,13 +25,20 @@ export async function loadDependency(
   FHIRDefs: FHIRDefinitions,
   cachePath: string = path.join(os.homedir(), '.fhir', 'packages')
 ): Promise<FHIRDefinitions> {
+  if (/(?=.*[A-Z])/.test(packageName)) {
+    logger.warn(
+      `${packageName} contains upper-case characters, which is discouraged. SUSHI will download ${packageName.toLowerCase()}#${version} if the package is not found in local cache.`
+    );
+    packageName = packageName.toLowerCase();
+  }
   let fullPackageName = `${packageName}#${version}`;
   let loadPath = path.join(cachePath, fullPackageName, 'package');
   let loadedPackage: string;
 
   // First, try to load the package from the local cache
   logger.info(`Checking local cache for ${fullPackageName}...`);
   loadedPackage = loadFromPath(loadPath, fullPackageName, FHIRDefs);
+
```

Comment:
Typo?

Reply:
Yes! I had initially added some code there but I realized it wasn't needed so I yanked it out. Addressed in commit `855cd26`.